src/script/lua_api/l_noise.cpp

```diff
@@ -437,7 +437,7 @@ int LuaPseudoRandom::create_object(lua_State *L)
 {
 	NO_MAP_LOCK_REQUIRED;
 
-	u64 seed = luaL_checknumber(L, 1);
+	u64 seed = read<float>(L, 1);
```

Comment:
Precision loss due to float conversion in this function.

Reply:
oops i missed that, i need to revert this change